src/main/java/org/mockito/invocation/Location.java

```diff
@@ -4,12 +4,9 @@
  */
 package org.mockito.invocation;
 
-import org.mockito.NotExtensible;
-
 /**
  * Describes the location of something in the source code.
  */
-@NotExtensible
```

Comment:
I don't think we want to make this part of an extension point for external users. E.g. while Mockito is allowed to implement such interfaces, we don't want external users to implement them. Therefore, let's remove this here and create a utility function that can create a Location based on a `SourceFile`?